### PR TITLE
Upgraded plugin to 2.0.3, nightly to RC8 (bug fixes)

### PR DESCRIPTION
**Pull request by helena**
Tested `akka-actor/scalafmtAll` to be sure no format changes were made.

### CODE REVIEW

File: .scalafmt.conf [hunk 1]
```diff
@@ -1,4 +1,4 @@
-version = 2.0.0-RC7
+version = 2.0.0-RC8
```
Comment (raboof) [inline .scalafmt.conf]: Can't this be `2.0.0` now?
```suggestion
version = 2.0.0
```

**Review by raboof — APPROVED** [review]
Once CI agrees ;)

**Comment by helena** [comment]
CI is happy now, weird travis cache issue.

File: .scalafmt.conf (hunk 1 above)
Comment (helena) [reply .scalafmt.conf]: I encountered an issue when I tried that. Let me retry to see what it was.

Comment (helena) [reply .scalafmt.conf]: @raboof this is the error if you also update the .scalafmt.conf file, which makes sense, these are different - the conf file and the sbt plugin, e.g. https://github.com/scalameta/sbt-scalafmt/blob/master/.scalafmt.conf#L1

```
[error] failed to resolve Scalafmt version '2.0.3': .../akka/.scalafmt.conf
```

I think it is fine for `plugins.sbt` with [the latest release version](https://github.com/scalameta/sbt-scalafmt/releases)

**Review by johanandren — APPROVED** [review]

**Pull request merged**